app/css/custom.scss

```diff
@@ -6,25 +36,29 @@ $blue: #225be2;
 .full-height {
   height: 100%;
 }
-aside {
-  .sidebar {
-    background-color: rgba(0, 0, 0, 0.09);
-    padding: 20px;
-    margin-right: -20px;
-    margin-top: -20px;
-  }
-  .trending-link {
-    &.item {
-      display: flex;
+.main {
```

Comment:
:yellow_heart: to avoid ambiguous code won't this be better written as aside.main?